src/gui/sdlmain.cpp

```diff
@@ -212,16 +220,17 @@ enum PRIORITY_LEVELS {
 	PRIORITY_LEVEL_HIGHEST
 };
 
-
 struct SDL_Block {
 	bool inited;
 	bool active;							//If this isn't set don't draw
 	bool updating;
 	bool update_display_contents;
 	bool resizing_window;
+	OutKind kind;
```

Comment:
> 'kind' is too generic for sdl scope (and not descriptive). Assuming this will be updated once we conclude the enum naming discussion.

Of course. It shall be named after the enum, `scalingMode` or `scalingType` &c.

Reply:
For now, I have renamed `OutKind` into `ScalingMode`, and the `kind` member into `scaling_mode` (by analogy with other lowercase members).